app/helpers/uploaded_evidence_helper.rb

```diff
@@ -0,0 +1,32 @@
+module UploadedEvidenceHelper
+  def evidence_message(required_documents, legal_aid_application, evidence_type_translation)
+    if required_documents.size == 1
+      single_evidence_message(required_documents.first, legal_aid_application, evidence_type_translation)
+    else
+      multiple_evidence_message(required_documents, legal_aid_application)
+    end
+  end
+
+private
+
+  def single_evidence_message(evidence, legal_aid_application, evidence_type_translation)
+    content_tag(:div, class: "govuk-body") do
+      if legal_aid_application.section_8_proceedings?
+        t(".section_8_evidence")
```

Comment:
using relative translation lookups outside of views, as here, could cause future headaches. Consider hardcoded translation paths or passing of arguments